recipes/andreasbuhr-cppcoro/all/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.0)
+project(cppcoro CXX)
```

Conflicts https://github.com/andreasbuhr/cppcoro/blob/7cc9433436fe8f2482138019cfaafce8e1d7a896/CMakeLists.txt#L2

```suggestion
project(cmake_wrapper CXX)
```